core/services/pipeline/task.eth_tx_test.go

```diff
@@ -291,6 +298,43 @@ func TestETHTxTask(t *testing.T) {
 			},
 			nil, nil, "", pipeline.RunInfo{},
 		},
+		{
+			"happy (missing gasLimit takes spec defined value)",
```

Comment:
do you mean "missing spec-defined gasLimit takes global value"?

Reply:
No, this actually means if the task has missing gasLimit parameter, then it takes gasLimit from job spec (which is defined in this test in the spec).
Note that we also have "happy (missing gasLimit takes config default)" test, that tests pretty much the same, but job spec does not override gasLimit and we take it from the env var.